mods/cnc/maps/nod06a/nod06a.lua

```diff
@@ -0,0 +1,223 @@
+	end
+end
+
+Atk2TriggerFunction = function()
+	if not Atk2Switch then
+		for type, count in pairs({ ['mtnk'] = 1, ['jeep'] = 1 }) do
+			MyActors = Utils.Take(count, GDI.GetActorsByType(type))
+			Utils.Do(MyActors, function(actor)
+				MovementAndHunt(actor, Gdi6Waypoints)
+			end)
+		end
+		Atk2Switch = true
+	end
+end
+
+Atk3TriggerFunction = function()
+	if not Atk3Switch then
+		local list = Nod.GetGroundAttackers()
+		AirstrikeProxy = Actor.Create("airstrike.proxy", true, { Owner = GDI })
+		AirstrikeProxy.SendAirstrike(list[DateTime.GameTime % #list + 1].CenterPosition, false, 256 - 28)
+		AirstrikeProxy.Destroy()
+		Atk3Switch = true
+	end
+end
+
+Chn1TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn1Units, Chn1Waypoints, nil,
+	function(transport, cargo)
+		transport.UnloadPassengers()
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end,
+	function(transport)
+		transport.Move(waypoint14.Location)
+		transport.Destroy()
+	end)
+end
+
+Chn2TriggerFunction = function()
+	Reinforcements.ReinforceWithTransport(GDI, 'tran', Chn2Units, Chn2Waypoints, nil,
+	function(transport, cargo)
+		transport.UnloadPassengers()
+		Utils.Do(cargo, function(actor)
+			IdleHunt(actor)
+		end)
+	end,
+	function(transport)
+		transport.Move(waypoint14.Location)
+		transport.Destroy()
+	end)
+end
+
+Obj2TriggerFunction = function()
+	Nod.MarkCompletedObjective(NodObjective2)
+	Reinforcements.Reinforce(Nod, Obj2Units, { Obj2UnitsEntry.Location, waypoint13.Location }, 15)
+end
+
+MovementAndHunt = function(unit, waypoints)
+	if unit ~= nil then
+		Utils.Do(waypoints, function(waypoint)
+			unit.AttackMove(waypoint.Location)
+		end)
+		IdleHunt(unit)
+	end
+end
+
+InserNodUnits = function()
+	Reinforcements.Reinforce(Nod, NodStartUnitsLeft, { UnitsEntryLeft.Location, UnitsRallyLeft.Location }, 15)
+	Reinforcements.Reinforce(Nod, NodStartUnitsRight, { UnitsEntryRight.Location, UnitsRallyRight.Location }, 15)
+end
+
+WorldLoaded = function()
+	GDI = Player.GetPlayer("GDI")
+	Nod = Player.GetPlayer("Nod")
+	
+	Trigger.OnObjectiveAdded(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "New " .. string.lower(p.GetObjectiveType(id)) .. " objective")
+	end)
+
+	Trigger.OnObjectiveCompleted(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective completed")
+	end)
+
+	Trigger.OnObjectiveFailed(Nod, function(p, id)
+		Media.DisplayMessage(p.GetObjectiveDescription(id), "Objective failed")
+	end)
+
+	Trigger.OnPlayerWon(Nod, function()
+		Media.PlaySpeechNotification(Nod, "Win")
+	end)
+
+	Trigger.OnPlayerLost(Nod, function()
+		Media.PlaySpeechNotification(Nod, "Lose")
+	end)
+
+	NodObjective1 = Nod.AddPrimaryObjective("Steal the GDI nuclear detonator")
+	NodObjective2 = Nod.AddSecondaryObjective("Kill the GDI supporter and destroy their houses in the village")
+
+	GDIObjective = GDI.AddPrimaryObjective("Stop the Nod taskforce from escaping with the detonator")
```

Comment:
Same here. It has to be "task force" as http://en.pons.com/translate/german-english/Taskforce is actually a German Anglicism.